core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -1259,66 +1268,86 @@
+        </xs:documentation>
       </xs:annotation>
     </xs:attribute>
     <xs:attribute name="strategy" type="tns:eviction-strategy">
       <xs:annotation>
-        <xs:documentation>The eviction strategy to use. This determines if eviction is even enabled or if it
-        has a different variant.</xs:documentation>
+        <xs:documentation>
+          Specifies a strategy for evicting cache entries. The default strategy
+          is NONE so you must specify a strategy for eviction to take place.
+        </xs:documentation>
       </xs:annotation>
     </xs:attribute>
   </xs:complexType>
 
   <xs:complexType name="memory-off-heap">
     <xs:attribute name="size" type="xs:long" default="-1">
       <xs:annotation>
-        <xs:documentation>The size of the eviction cache as a long.  If the configured type is COUNT, this will be
-          how many entries can be stored.  If the configured type is MEMORY, this will be how much memory
-          in bytes can be stored.</xs:documentation>
+        <xs:documentation>
+          Defines the size of the data container as a long. Eviction occurs
+          either when the number of entries or amount of memory exceeds the
+          size.
+      </xs:documentation>
       </xs:annotation>
     </xs:attribute>
     <xs:attribute name="eviction" type="tns:eviction-type">
       <xs:annotation>
-        <xs:documentation>The eviction type to use whether it is COUNT or MEMORY.  COUNT will limit the cache based on
-          the number of entries.  MEMORY will limit the cache by how much memory the entries use</xs:documentation>
+        <xs:documentation>
+          Specifies whether eviction is based on the number of entries or the
+          amount of memory used.
+        </xs:documentation>
       </xs:annotation>
     </xs:attribute>
     <xs:attribute name="strategy" type="tns:eviction-strategy">
       <xs:annotation>
-        <xs:documentation>The eviction strategy to use. This determines if eviction is even enabled or if it
-          has a different variant.</xs:documentation>
+        <xs:documentation>
+          Specifies a strategy for evicting cache entries. The default strategy
+          is NONE so you must specify a strategy for eviction to take place.
+        </xs:documentation>
       </xs:annotation>
     </xs:attribute>
     <xs:attribute name="address-count" type="xs:int" default="1048576">
       <xs:annotation>
-        <xs:documentation>How many address pointers to use. This number will be rounded up to a power of two.
-          For optimal performance you will want more address pointers than you expect to have entries.  This is similar
-          to the size of an array backing a hash map.  Without collisions lookups and writes will be constant time.
-          Each pointer will take up 8 bytes of memory thus the default will use 8 MB of off-heap memory.
+        <xs:documentation>
+          Specifies the number of pointers that are available in the hash map,
```

Comment:
This setting is no longer used as of 10.0. I am actually removing it in another PR.

Reply:
@wburns hey, should we keep this in the doc PR for 10.1.x and remove in this doc PR?

Reply:
Well technically as of 10.0 this configuration option does absolutely nothing.

Reply:
ok, let's just drop it all together.

Reply:
Sounds good to me.